profile/README.md

```diff
@@ -7,7 +7,7 @@ We're glad you're here! This is where we host all of our Official[^1] SDKs for t
 
 Currently, GitHub maintains SDKs for the following languages/frameworks/platforms:
 - [JavaScript](https://github.com/octokit?q=&type=all&language=javascript&sort=) / [TypeScript](https://github.com/octokit?q=&type=all&language=typescript&sort=)
-- [C# / .NET](https://github.com/octokit?q=&type=all&language=c%23&sort=)
+- [C# / .NET](https://github.com/octokit/octokit.net)
```

Comment:
There is more than one repository. The original link is a filter for repositories, but it's easy to miss because the repositories are only shown at the bottom of the page. I think it works if we add the `#org-profile-repositories` anchor.

```suggestion
- [C# / .NET](https://github.com/octokit?language=c%23#org-profile-repositories)
```

We can also remove `q`, `type`, and `sort` query parameters. Could you update the other links? That'd be fantastic

Reply:
Hi @gr2m sure. Sounds a good way.